CMakeLists.txt

```
@@ -1719,7 +1719,31 @@ endif()
 
 # Generate and use MCUboot related artifacts as needed.
 if(CONFIG_BOOTLOADER_MCUBOOT)
-  include(${CMAKE_CURRENT_LIST_DIR}/cmake/mcuboot.cmake)
+  get_property(signing_script_set
+    GLOBAL PROPERTY
+    signing_script SET
+    )
+
+  # If the signing_script property is not set, use the default MCUboot one included with Zephyr
+  if(${signing_script_set} EQUAL "0")
```

if keeping the `get_property(... SET)`, then the test should test against a number, but just test like `if(${signing_script_set})`, cause CMake only guarantees a boolean result, not a `0` or `1`.
```suggestion
if(${signing_script_set})
```

> If the SET option is given the variable is set to a boolean value indicating whether the property has been set.

---

Have updated to just use if()